src/shadowbox/model/access_key.ts

```diff
@@ -27,19 +27,19 @@ export interface ProxyParams {
   readonly password: string;
 }
 
-// Parameters needed to limit access key data usage over a sliding window.
-export interface AccessKeyQuota {
+// Parameters needed to limit access key data usage over a sliding timeframe.
+export interface AccessKeyLimit {
   // The allowed metered data transfer measured in bytes.
   readonly data: {bytes: number};
-  // The sliding window size in hours.
-  readonly window: {hours: number};
+  // The sliding timeframe size in hours.
+  readonly timeframe: {hours: number};
 }
 
-// Parameters needed to enforce an access key data transfer quota.
-export interface AccessKeyQuotaUsage {
-  // Data transfer quota on this access key.
-  readonly quota: AccessKeyQuota;
-  // Data transferred by this access key over the quota window.
+// Parameters needed to enforce an access key data transfer limit.
+export interface AccessKeyLimitUsage {
```

Comment:
This reads a little off. Sounds like a verb, while it should be a noun. I get it's a pair of nouns, but I wonder if we can phrase it a different way. Maybe `AccessKeyUsageLimit` or `AccessKeyLimitAndUsage`?

Reply:
With the rename to "AccesKeyDataLimitUsage" this no longer seems like a problem:
"(DataLimit)Usage" reads more naturally than "Data(LimitUsage)". Wdyt?